swift/README.md

```diff
@@ -6,6 +6,27 @@ We also follow the [Swift API design guidelines](https://swift.org/documentation
 
 Our overarching goals are conciseness, readability, and simplicity.
 
+## Linting
+
+Included is a default Mobify Swift Style Guide SwiftLint configuration,`.swiftlint.yml`.
+
+You can install [SwiftLint](https://github.com/realm/SwiftLint) by command `brew install swiftlint`
+
+### Integration
+Integrate SwiftLint into Xcode to get warnings and errors displayed. You can add a new "Run Script" with:
+
+```bash
+if which swiftlint >/dev/null; then
+  swiftlint
```

Comment:
No, the swiftlint yaml file lives in the root of your iOS project. To point it to the code style lint file we would have to... make a symlink?